guardian/SDL/.gdnbaselines

```diff
@@ -0,0 +1,26 @@
+{
+  "version": "latest",
```

Comment:
>version": "latest [](http://example.com/codeflow?start=3&length=17)

@payton2022 , please, pull the last commits and remove this file. We won't need it anymore since  Chris removed the "secrets" from the code.